dubbo-admin-ui/src/components/test/ServiceTest.vue

```diff
@@ -230,7 +237,7 @@
     mounted () {
       this.$store.dispatch('loadServiceItems')
       let query = this.$route.query
-      this.filter = query['service']
+      this.filter = query['service'] ? query['service'] : ''
```

Comment:
`query['service'] ? query['service'] : ''` can be replaced with a simpler way `query['service'] || ''`